Apps/phwinrm/parse_callbacks.py

```diff
@@ -100,22 +112,43 @@ def list_processes(action_result, response):
         )
 
     output = response.std_out
-    lines = output.splitlines()
-    for line in lines[2:]:
-        process = {}
+    lines = [y for y in [i for i in output.splitlines() if i] if not y.startswith('-----')]
+    column_headers = lines[0]
```

Comment:
is it guaranteed that `lines` will have at least one value? otherwise, can we gracefully handle the case where lines ends up empty?

Reply:
Fixed